challenges/Backend Challenge/app.py

```diff
@@ -0,0 +1,29 @@
+from flask import Flask
+import requests
+
+app = Flask(__name__)
+arr = []
+
+@app.route('/')
+def index():
+	return "Hello World!"
+
+@app.route('/data', methods = ['GET'])
+def sortnumbers():
+	if len(arr) == 0:
+		return "Number list is empty"
+	arr.sort()
```

Comment:
You decided to sort the list in the GET request, which works.  I have a question.  Assume our client will POST a list of numbers once, and then GET that same list of numbers millions of times every day.  How would this change your implementation to minimize computation?  You don't need to write any more code, just respond with a short description of how you would re-design this solution to handle the scenario I described.

Reply:
@CharlesAHunt : If the user is just going to POST the number list once, I could sort it then itself, and for every GET call, simply return the sorted list as many as times required in the necessary format!